src/graphql/account.rs

```diff
@@ -28,6 +28,11 @@ use crate::graphql::query_with_constraints;
 pub struct SignedInAccount {
     username: String,
     expire_times: Vec<DateTime<Utc>>,
+    signin_times: Vec<DateTime<Utc>>,
```

Comment:
`SignedInAccount` is a part of the GraphQL API, and thus these changes need to be noted in CHANGELOG.